### PR TITLE
refactor constants

### PR DESCRIPTION
**Pull request by moonso**
This PR refactors the huge constants file into a directory with multiple contant files

## Review
- [x] code approved by @barrystokman 
- [x] "Merge and deploy" approved by @barrystokman 
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/constants/constants.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+"""Constans for cg"""
```
Comment (barrystokman) [inline cg/constants/constants.py:1]:
```suggestion
"""Constants for cg"""
```

**Review by barrystokman — APPROVED** [review]
This cleans up nicely :+1:

**Review by patrikgrenfeldt — APPROVED** [review]
Great work!

**Comment by moonso** [comment]
Skärmavbild 2020-10-27 kl  09 53 27

Merged and deployed on hasta

**Pull request merged**